keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepositoryTests.kt

```diff
@@ -177,70 +181,116 @@ abstract class ResourceRepositoryTests<T : ResourceRepository> : JUnit5Minutests
         }
       }
 
-      context("updating the state of the resource") {
-        context("an event that should be persisted in history") {
+      context("updating the event history of the resource") {
```

Comment:
There was never anything within this `context` about updating a resource, per se, but rather about updating the event history of the resource, so I reworded it and the nested contexts and tests accordingly.